spec/abstractops.html

```diff
@@ -1843,7 +1845,7 @@ <h1>
           </tr>
           <tr>
             <td>*"monthCode"*</td>
-            <td>~ToString~</td>
+            <td>~ToPrimitiveAndRequireString~</td>
```

Comment:
This is an odd name; perhaps just “ToPrimitiveString”?

Reply:
I'm not a fan of RequireString because that name masks the coercion that may happen.

I'm also not a fan of ToPrimitiveString because it's not clear (to ma at least!) what's the difference between ToString and ToPrimitiveString, and even might suggest that there's some special kind of String that's not a primitive.

So if OK with you guys, in the interest of getting this normative PR merged soon so we can get implementers working on Temporal in earnest, I'd prefer to keep the current long name (which I agree is a bit awkward) for now. We can always editorially rename it later if we can figure out a better name. 

So I'll resolve this thread now, but feel free to un-resolve and comment if you disagree... or if you have a better name idea!